app/src/components/staking/DelegationsOverview.vue

```diff
@@ -13,11 +13,35 @@
       </div>
       <div v-else-if="delegations.length > 0">
         <h1>Your Validators</h1>
+        <h2 v-if="delegations.find(({ active }) => active === 'INACTIVE')">
+          Active Delegations
+        </h2>
         <TableValidators
-          :validators="delegations.map(({ validator }) => validator)"
-          :delegations="delegations"
+          :validators="
+            delegations
+              .filter(({ active }) => active === 'ACTIVE')
+              .map(({ validator }) => validator)
+          "
+          :delegations="delegations.filter(({ active }) => active === 'ACTIVE')"
           show-on-mobile="expectedReturns"
         />
+        <div
+          v-if="delegations.find(({ active }) => active === 'INACTIVE')"
+          class="table-subcontainer"
+        >
+          <h2>Inactive delegations</h2>
```

Comment:
what are inactive delegations? why not use the undelegation table?

Reply:
no, inactive delegations are delegations that are on the waitlist (one session takes about one day and not all of them will get through even within that time).

So, for example, when you delegate your delegations are also inactive until that delegation goes through. And you don't want that to be reflected as an undelegation, that would be too confusing.

Actually they are delegations on the waitlist

Reply:
But I think it is even more subtle than that. Polkadotjs distinguishes between Waiting nominations and Inactive nominations, although I believe both of them we are getting them from the inactive nominations query